GolfDiva-LaHabraInfo

```diff
@@ -0,0 +1,8 @@
+** La Habra
```

Comment:
La Habra is 10 Miles north of Disneyland. It is the north/west most corner of Orange County